_static/api-examples.md

```diff
@@ -152,7 +152,7 @@ You can sign a manifest with either the e-Manifest ID:
 ```bash
 curl -i -X POST -H 'Content-Type: application/json' \
   --data @sign-manifest-emanifestid.json \
-  https://e-manifest.18f.gov/api/0.1/manifest/sign
+  https://e-manifest.18f.gov/api/v1/manifests/:manifest_id/sign
```

Comment:
I like that! more typing but I will jut have to reply on autocomplete :smile_cat: